build/assets/scripts/high-performance-hooks.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+pid=$(jq '.pid' /dev/stdin)
+mask=$1
+ns=$(ip netns identify "${pid}")
+
+[[ -n "${pid}" ]] && [[ -n "${mask}" ]] && [[ -n "${ns}" ]] || exit 0
```

Comment:
is it worth to exit early if mask exists and it is equal to "0" (= rps disabled)?

Reply:
in the future we may want to disable rps... and the default value is ) anyway, I would leave it as is

Reply:
works for me!